src/test/java/com/datadog/api/v1/client/api/AwsIntegrationApiTest.java

```diff
@@ -86,8 +86,9 @@ public void assertAccountIn(AWSAccount accountToAssert, List<AWSAccount> account
     public void createAWSAccountTest() throws TestUtils.RetryException {
         //Test Creating an AWS Account with just the account_id and role_name
         AWSAccount awsAccount = new AWSAccount();
-        awsAccount.setAccountId(String.format("java_%07d", (now.toInstant().toEpochMilli()) % 10000000));
-        awsAccount.setRoleName("java_testRoleName");
+        String uniqueName = getUniqueEntityName();
+        awsAccount.setAccountId(uniqueName.substring(0, 12));
```

Comment:
I'd rather have a random 12 char string here, GUID like, since we can identify the origin with the role name, we don't really care about what the accountID is. Better to be sure not to have conflicts.

Reply:
We need to make sure that the generation is stable so we can (a) use current time as a seed (b) store a random seed next to cassttes.

Reply:
I can just revert to what we had before, that worked as well (and we can keep the whole unique name as the `roleName`)

Reply:
Yeah what we had before should be random enough